### PR TITLE
Introduction

### PR DESCRIPTION
**Pull request by Alp4ga**
- Create new file Handbook.md
- Write introduction
- Write all part of the documents with Table of contents

### CODE REVIEW

File: Handbook.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+## Introduction
+
+### 1.1 Welcome
+As a new employee, you will receive a written handbook of best practices.
+This document outlines the general best pratices you must follow during the development of a project. Please read it carefully.
+
+### 1.2 Purpose of this Handbook
+This Employee Handbook is designed to provide you with information about working
+conditions and pratices you must follow.
```
Comment (Az8th) [inline Handbook.md]:
```suggestion
conditions and practices you must follow.
```

File: Handbook.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+You will learn about Task estimation in scrum process. The coding standars you must follow to help your co-workings understanding your code.
```
Comment (Az8th) [inline Handbook.md]:
```suggestion
You will learn about Task estimation in scrum process and the coding standards you must follow to help your co-workers understand your code.
```

**Review by Az8th — CHANGES_REQUESTED** [review]
Rewiew : content ok. Two typos to fix before merging

**Review by Alp4ga — COMMENTED** [review]
The new typo seems mutch better

**Review by Az8th — APPROVED** [review]

**Review by BetaTuX — APPROVED** [review]

**Pull request merged**